lang_python/parsing/AST_python.ml

```diff
@@ -352,7 +352,7 @@ type stmt =
   | Break of tok | Continue of tok
   | Pass of tok
 
-  | Raise of tok * (expr * expr option (* from *)) option
+  | Raise of tok * (expr * expr option (* from *)) option * expr option (* arguments *) * expr option (* location *)
```

Comment:
I would have added many a python2: comment saying last 2 args are only in python2

Reply:
Or even made an alternate RaisePython2 of ...